app/src/pages/inside/projectSettingsPageContainer/content/integrations/integrationsList/integrationInfo/integrationCollection/integrationCollection.jsx

```diff
@@ -23,10 +23,12 @@ import styles from './integrationCollection.scss';
 const cx = classNames.bind(styles);
 
 export const IntegrationCollection = ({ items, disabled, openIntegration }) => {
+  const sortedItems = items.sort((a, b) => b.creationDate - a.creationDate);
```

Comment:
Do we really need to sort that array of integrations on every re-renders?

Could it be better to create an effect? Which runs on componentdidmount, and sort that array inside that effect. I suggest you use the useLayoutEffect hook here.